testsuite/openshift/limitador.py

```diff
@@ -0,0 +1,16 @@
+"""Limitador CR object"""
+
+from openshift_client import selector
+
+from testsuite.openshift import CustomResource
+from testsuite.openshift.deployment import Deployment
+
+
+class LimitadorCR(CustomResource):
+    """Represents Limitador CR objects"""
+
+    @property
+    def deployment(self) -> Deployment:
```

Comment:
You have two way of achieving the same thing based on deployment type:
Standalone: `limitador_cr.deployment`
Kuadrant: `kuadrant_cr.limitador_deployment`

It would be good to unify path to obtain the same thing regardless of deployment

Reply:
I'm working on this in another commit/PR